src/pages/heeyeolKim/Main/Main.js

```diff
@@ -0,0 +1,318 @@
+                                    <div>
+                                        <button 
+                                        className="uploadComment"
+                                        onClick = {this.uploadNewComment}
+                                        >
+                                            <a href="#!">
+                                                <span>게시</span>
+                                            </a>
+                                        </button>
+                                    </div>
+                                </div>
+                            </section>
+                        </article>
+                    </main>
+
+                    <aside>
+                        <div className="asideContainer">
+                            <div className="asideProfile">
+                                <div className="asideProfileContainer">
+                                    <div className="asideProfileImage">
+                                        <img src={story1} alt=""/>
+                                    </div>
+                                    <div className="asideProfilename">
+                                        <ul>
+                                            <li>hereizhere</li>
+                                            <li>김희열</li>
+                                        </ul>
+                                    </div>
+                                </div>
+                                <div className="asideProfileChange">
+                                    전환
+                                </div>
+                            </div>
+                            <ul className="asideRecomment">
+                                <li className="asideRecommentForMe">회원님을 위한 추천</li>
+                                <li className="asideRecommentAll">모두 보기</li>
+                            </ul>
+                            <ul className="recommentList">
+                                <ul className="recommentListLeft">
+                                    <li className="recommentListImage">
+                                        <img src={p1} alt=""/>
+                                    </li>
+                                    <li>
+                                        <ul className="recommentListCenter">
+                                            <li>monday</li>
+                                            <li>회원님을 팔로우합니다.</li>
+                                        </ul>
+                                    </li>
+                                </ul>
+                                <li className="recommentListFollow">팔로우</li>
+                            </ul>
+                            <ul className="recommentList">
+                                <ul className="recommentListLeft">
+                                    <li className="recommentListImage">
+                                        <img src={p2} alt=""/>
```

Comment:
```suggestion
                                        <img src={p2} alt="alt메세지를 입력!"/>
```
신영님이 alt메세지는 항상 쓰는게 좋다고 하신 것 같습니다!

Reply:
이유는 뭘까요? @ljh95 @Kim-Heeyeol

Reply:
1) 이미지를 볼 수 없는 환경에서 이미지에 대한 설명을 하거나, 
2) alt 문자열도 메타정보 중 하나이기 때문에 검색엔진최적화를 통해 검색봇(?)이 컨텐츠를 이해하기 위해서
입니다!!